TapCounter/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  TapCounter
+//
+//  Created by Alice Aponasko on 1/24/16.
+//  Copyright © 2016 justforfun. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
Also, this is a one view controller app. How do you want to rename it? HomeViewController? MainViewController?